admin-ui/app/redux/features/licenseSlice.js

```diff
@@ -63,6 +64,10 @@ const licenseSlice = createSlice({
     },
     generateTrialLicenseResponse: (state) => {
       state.generatingTrialKey = false
+    },
+    handleApiTimeout: (state, action) => {
```

Comment:
this is a common function applied to all requests so should not be placed in `licenseSlice.js`